### PR TITLE
Catbeasts and Cluwnes (and Meeseeks) can no longer suicide

### PR DESCRIPTION
**Pull request by tortellinitony**
:cl: Tortellini Tony
fix: Catbeasts and Cluwnes can no longer commit suicide and are now damned to live a life of suffering with no easy way out.
/:cl:

### CODE REVIEW

**Review by ghost — DISMISSED** [review]
Code OK, needs @HippieStation/head approval

**Comment by PopNotes** [comment]
holy shit tony actually did something funny for once +1

**Comment by ghost** [comment]
Disable the `succumb` and `ghost` verb as well

**Comment by dreamcrab** [comment]
run it fergie ferg +7777

**Comment by Spacedong** [comment]
epic

**Comment by Spacedong** [comment]
can I suggest making a special message that appears in chat when you try to do it

**Comment by PopNotes** [comment]
not mcmodular scree

**Comment by Ragicil** [comment]
Might want to make it so that they, somehow, cannot leave via airlock nor be able to inflict damage upon themselves.
I envision a future where disgusting cats beg assistants for the sweet release of death. Only to be captured and held for  _e x p e r i m e n t a t i o n_

File: code/modules/mob/dead/observer/observer.dm [hunk 1]
```diff
@@ -268,6 +268,8 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 	set name = "Ghost"
 	set desc = "Relinquish your life and enter the land of the dead."
 
+	if(!canGhost())
```
Comment (ghost) [inline code/modules/mob/dead/observer/observer.dm]: You've made changes to /tg/ files, please mark these changes.

**All modifications to non-hippie files should be marked.**

- Multi line changes start with `// hippie start` and end with `// hippie end`
- You can put a messages with a change if it isn't obvious, like this: `// hippie start - reason`
  - Should generally be about the reason the change was made, what it was before, or what the change is
  - Multi-line messages should start with `// hippie start` and use `/* Multi line message here */` for the message itself
- Single line changes should have `// hippie` or `// hippie - reason`

If you need to mirror a file, or function into a hippie-specific file, please leave behind a comment stating where it went.

```
// hippie start - Mirrored this function in <file> for <reason>
bunch of shitcode here
// hippie stop
```

Once you mirror a file, please follow the above for marking your changes, this way we know what needs to be updated when a file has been mirrored.

File: code/modules/mob/living/living.dm [hunk 2]
```diff
@@ -267,6 +267,8 @@
 
 /mob/living/verb/succumb(whispered as null)
 	set hidden = 1
+	if(!canSuccumb())
```
Comment (ghost) [inline code/modules/mob/living/living.dm]: You've made changes to /tg/ files, please mark these changes.

**All modifications to non-hippie files should be marked.**

- Multi line changes start with `// hippie start` and end with `// hippie end`
- You can put a messages with a change if it isn't obvious, like this: `// hippie start - reason`
  - Should generally be about the reason the change was made, what it was before, or what the change is
  - Multi-line messages should start with `// hippie start` and use `/* Multi line message here */` for the message itself
- Single line changes should have `// hippie` or `// hippie - reason`

If you need to mirror a file, or function into a hippie-specific file, please leave behind a comment stating where it went.

```
// hippie start - Mirrored this function in <file> for <reason>
bunch of shitcode here
// hippie stop
```

Once you mirror a file, please follow the above for marking your changes, this way we know what needs to be updated when a file has been mirrored.

**Review by ghost — DISMISSED** [review]

**Review by ghost — DISMISSED** [review]

**Comment by ghost** [comment]
build this

**Review by ghost — APPROVED** [review]

**Comment by KayeArray** [comment]
+1

**Comment by bardofrage** [comment]
+1

**Comment by Swagile** [comment]
i mean its ok, but people will find a way around this anyway unless we make cat beasts and cluwnes invulnerable, which defeats the entire purpose of cat beasts and cluwnes

**Comment by PopNotes** [comment]
now has a total of +3 and is very old, mergin'

**Pull request merged**